packages/design/components/Image/style/index.less

```
@@ -0,0 +1,18 @@
+@loadingColor: #EDE9E9;
+@outlineColor: rgba(255, 255, 255, 0.4);
+@boxShadowColor: rgba(237, 233, 233, 0.8);
+
+.bgm-image {
+  display: inline-block;
+  overflow: hidden;
+  background: @loadingColor;
+
```

object-fit 默认设置成 cover 会不会比较好？
```suggestion
object-fit: cover;

```